layers/meta-balena-beaglebone/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -0,0 +1,18 @@
+FILESEXTRAPATHS:append := ":${THISDIR}/${PN}"
```

Comment:
We should leave a note on the reason for disabling the watchdog

Reply:
I have already added a line to the commit message: https://github.com/balena-os/balena-beaglebone/pull/686/commits/fc7769ef2aa9bd76e51e330f73499dc516468039 (reproduced below):

"Also, disabled the hardware watchdog (due an ongoing open issue, whose fix is currently being tested). Add the "test_watchdog" package to the kernel package (for easy verification of the patch, when it become available)."

Reply:
It doesn't hurt to leave a comment in the file because it's quicker for someone investigating a problem and looking at this file on a remote device to understand the reason behind the configuration. Otherwise he would have to go find the device integration repository, search for the yocto package as well as for the layer which provides/modifies it then look in the git history for it, to finally understand why. cc @floion